website/components/live-example/Knobs.tsx

```diff
@@ -13,11 +14,12 @@ const areaIdAllocator = IdAllocator.create('area');
 const numberIdAllocator = IdAllocator.create('number');
 const selectIdAllocator = IdAllocator.create('select');
 
+const knobsWidth = 318; // totalWidth (700px) - padding on both sides (24px on each side) / 2
```

Comment:
this doesn't add up. (700 - 24 * 2) / 2 = 326

Reply:
Good catch 😅